diamond/bin/diamond

```diff
@@ -23,8 +23,14 @@ import traceback
 import string
 
 import gi
+
 gi.require_version('Gtk', '3.0')
-gi.require_version('GtkSource', '3.0')
+
+try:
+  gi.require_version('GtkSource', '4')
```

Comment:
On my machine, `gi.require_version('GtkSource', '3')' yields a ValueError, but `gi.require_version('GtkSource', '3.0')' does not, so I am wondering if you need `4.0` here instead of `4'.

Reply:
Updated but not yet checked on relevant systems - will aim to do so today. Thanks for the catch!

Reply:
I tested 4.0 as well and that didn't work. It seems it needs to be 4 for whatever reason but 3 has to be 3.0 ...